packages/core/src/secret/__tests__/secret-hash.test.ts

```
@@ -2,76 +1,432 @@
+import {
+  hash160,
```

ESLint

[error] hash160 not found in '../hash'

(import/named)